api/envoy/config/filter/http/compressor/v2/compressor.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.compressor.v2;
+
+option java_outer_classname = "CompressorProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.filter.http.compressor.v2";
+option go_package = "v2";
+
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Compressor]
+
+message Compressor {
+
+  // Minimum response length, in bytes, which will trigger compression. The default value is 30.
+  google.protobuf.UInt32Value content_length = 1 [(validate.rules).uint32.gte = 30];
+
+  // Set of strings that allows specifying which mime-types yield compression; e.g.,
+  // application/json, text/html, etc. When this field is not defined, compression will be applied
+  // to the following mime-types: "application/javascript", "application/json",
+  // "application/xhtml+xml", "image/svg+xml", "text/css", "text/html", "text/plain", "text/xml".
+  repeated string content_type = 2 [(validate.rules).repeated = {max_items: 50}];
```

Comment:
Why is there a maximum?

Reply:
Actually I can't find any reason why there should be a hard limit for the field.

@gsagula do you mind if I drop it?

Reply:
@gsagula ping

Reply:
@gsagula ping. I'll drop the limit in a couple of days if you don't mind.

Reply:
@rojkov Go ahead! I don't mind.